maven/JarAssembler.kt

```diff
@@ -0,0 +1,68 @@
+import picocli.CommandLine
+import picocli.CommandLine.Command
+import picocli.CommandLine.Option
+import java.io.BufferedInputStream
+import java.io.BufferedOutputStream
+import java.io.File
+import java.io.FileOutputStream
+import java.util.*
+import java.util.concurrent.Callable
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+import java.util.zip.ZipOutputStream
+import kotlin.system.exitProcess
+
+
+@Command(name = "pom_generator", mixinStandardHelpOptions = true)
```

Comment:
Is the `@Command`'s `name` supposed to be `jar-assembler` instead?

Reply:
@lolski correct; addressed in https://github.com/graknlabs/bazel-distribution/pull/293/commits/822ba2566e42ea7e15ed59e71c323113c3f9b4d5